tasks/main.yml

```diff
@@ -14,7 +14,34 @@
     # it.
     state: latest  # noqa 403
     update_cache: yes
-  when: ansible_os_family == "Debian"
+  when: >
+    ansible_os_family == "Debian" and
+    ansible_distribution_release != "kali-rolling"
+
+- name: Reinstall cryptsetup-initramfs so upgrade doesn't fail
+  apt:
+    name: 'cryptsetup-initramfs'
+    state: latest  # noqa 403
+    update_cache: yes
+  when: ansible_distribution_release == "kali-rolling"
+
+- name: apt-get update
+  shell: apt-get update  # noqa 303 305
+  args:
+    warn: false
+  when: ansible_distribution_release == "kali-rolling"
+  tags: molecule-idempotence-notest
```

Comment:
Is this some molecule-fu that tells the idempotence test to ignore for this step?  If so then there should be a comment explaining that.

Reply:
Yes, it is molecule-fu to ignore the idempotence test. Addressed in commit 9f51229

Reply:
Typo in "guarantee".